### PR TITLE
fix some  brackets and N/A in CHANGELOG-1.8

### PR DESCRIPTION
**Pull request by jianglingxia**
**What this PR does / why we need it**:
in CHANGELOG-1.8 Cluster Lifecycle and OpenStack place have redundant brackets 
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by jianglingxia** [comment]
/cc @wojtek-t
thanks!

File: CHANGELOG-1.8.md [hunk 1]
```diff
@@ -1106,7 +1106,7 @@ filename | sha256 hash
 * kubeadm: Use the release-1.8 branch by default ([#52085](https://github.com/kubernetes/kubernetes/pull/52085), [@luxas](https://github.com/luxas))
 * PersistentVolumeLabel admission controller is now deprecated. ([#52618](https://github.com/kubernetes/kubernetes/pull/52618), [@dims](https://github.com/dims))
 * Mark the LBaaS v1 of OpenStack cloud provider deprecated. ([#52821](https://github.com/kubernetes/kubernetes/pull/52821), [@FengyunPan](https://github.com/FengyunPan))
-* NONE ([#52819](https://github.com/kubernetes/kubernetes/pull/52819), [@verult](https://github.com/verult))
+* Fixed nil dereference in dynamic provisioning e2e tests ([#52819](https://github.com/kubernetes/kubernetes/pull/52819), [@verult](https://github.com/verult))
```
Comment (xiangpengzhao) [inline CHANGELOG-1.8.md]: This and the below two places don't need release notes, you can remove them (the original `NONE` lines) directly.

**Review by xiangpengzhao — CHANGES_REQUESTED** [review]

File: CHANGELOG-1.8.md (hunk 1 above)
Comment (jianglingxia) [reply CHANGELOG-1.8.md]: thanks，I have updated it!

**Comment by jianglingxia** [comment]
/assign @lavalamp
for /approve no-issue

**Pull request merged**